prime-router/src/main/kotlin/transport/RESTTransport.kt

```diff
@@ -562,17 +566,24 @@ class RESTTransport(private val httpClient: HttpClient? = null) : ITransport {
         private const val TIMEOUT = 120_000
 
         /** Our default Http Client, with an optional SSL context, and optional auth token */
-        private fun createDefaultHttpClient(jks: UserJksCredential?, accessToken: String?): HttpClient {
+        private fun createDefaultHttpClient(
+            jks: UserJksCredential?,
+            accessToken: String?,
+                                            restTransportInfo: RESTTransportType?,
+        ): HttpClient {
             return HttpClient(Apache) {
                 // installs logging into the call to post to the server
                 install(Logging) {
                     logger = io.ktor.client.plugins.logging.Logger.Companion.SIMPLE
                     level = LogLevel.INFO // LogLevel.INFO for prod, LogLevel.ALL to view full request
                 }
+
                 // not using Bearer Auth handler due to refresh token behavior
                 accessToken?.let {
+                    // Default to set Bearer prefix
+                    val authPrefix = restTransportInfo!!.headers["BearerToken"] ?: "Bearer"
```

Comment:
Slightly pedantic but let's call this authSchema since that's its name in the RFC.

Reply:
That sounds good, I will call it authSchema instead of authPrefix.